src/app/GitExtUtils/GitUI/Interops/User32/GWL.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace System;
+
+internal static partial class NativeMethods
+{
+    /// <summary>
+    ///  Window long values for <see cref="GetWindowLongPtrW(nint, GWL)"/>.
+    /// </summary>
+    public enum GWL : int
+    {
+        WNDPROC = (-4),
+        HWNDPARENT = (-8),
+        STYLE = (-16),
+        EXSTYLE = (-20),
+        ID = (-12),
+    }
```

Comment:
caused a warning
```suggestion
    internal enum GWL : int
    {
        WNDPROC = -4,
        HWNDPARENT = -8,
        STYLE = -16,
        EXSTYLE = -20,
        ID = -12,
    }
```

Reply:
A lot of places in documentation and on the internets has these negative numbers in parentheses, some kind of unwritten rule it seems.. maybe mimicking accountant style of denoting negative numbers..? Anyhow..